src/Bicep.Core.Samples/Files/InvalidOutputs_CRLF/Completions/decorators.json

```diff
@@ -1,152 +1,19 @@
-[

```

Comment:
Not sure why this is being removed...

Reply:
I think because this decorator (and the `decoratorsPlusNamespace` example) was previously not followed by any declaration in this file, the list of suggestions was previously giving the full list of decorators that are valid on **any** declaration.

Because you added a resource declaration below the decorator [here](https://github.com/Azure/bicep/blob/7d720e7c317a8401bb0a4e9da151f1f3035d8d69/src/Bicep.Core.Samples/Files/InvalidOutputs_CRLF/main.bicep#L139), the suggestions are being filtered down to only decorators that are appropriate on a resource declaration.

Reply:
Ok, I see. Shall I move it upwards?

Reply:
I think so, and probably just worth adding a comment like this one to explain why the decorators are at the end of the file so that it's clear to the next person editing this file: https://github.com/Azure/bicep/blob/187d4d2047dc83c69695ba79761f552bcb00c319/src/Bicep.Core.Samples/Files/InvalidExpressions_LF/main.bicep#L251

Reply:
Note here - deletion and removal is caused by CRLF file being check-in in the main branch, while my git did CRLF -> LF conversion.

Reply:
@anthony-c-martin - should I revert this to CRLF (and perhaps add .gitattribute to preserve line endings) or we're fine with json diagnostics are checked-in with LF?